pegasus/sites.v3/hourofcode.com/views/events/event_registration_form.haml

```diff
@@ -0,0 +1,29 @@
+:scss
+  .event-registration-form {
```

Comment:
out of curiousity, why put this inline instead of in a SCSS file?

Reply:
This was the only part of the page that needed custom styles and it felt like overkill to make a new stylesheet 😅